res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1042,243 +1014,142 @@ PioneerDDJSX.samplerVelocityVolume = function(channel, control, value, status, g
     }
 };
 
-PioneerDDJSX.parameterLeft = function(channel, control, value, status, group) {
-    var deck = PioneerDDJSX.channelGroups[group],
+PioneerDDJSX.changeParameters = function(group, ctrl, increment) {
+	var deck = PioneerDDJSX.channelGroups[group],
         index,
         offset = 0,
         samplerIndex = 0;
+        
+    //Roll Mode:
```

Comment:
Inconsistent spacing in this section, [use 4 spaces to indent](https://www.mixxx.org/wiki/doku.php/coding_guidelines#tabs_vs_spaces).